scripts/ci/build-installers.sh

```diff
@@ -44,6 +44,10 @@ if [ -z "$ARGV_OPERATING_SYSTEM" ] || [ -z "$ARGV_ARCHITECTURE" ]; then
   usage
 fi
 
+if [ "${GH_TOKEN:-false}" != "false" ]; then
```

Comment:
I guess an 'easy' way to fix that, would be to make `PUBLISH` a secure variable too, so that it's only set when `GH_TOKEN` is? ;-)

> I guess we can fix this by checking the opposite case: If PUBLISH is enabled, but there is no GH_TOKEN, we show a warning and unset PUBLISH

That also works :)